x.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import glob
+import os
+import pathlib
+import click
+import semver
```

Comment:
I think the use of packages in pypi might make it a little more difficult for users to use this script than before?

Reply:
It's possible to rewrite usage of `click` with `argparse` and `semver` with simple function, while I can see installing pypi libs is a well-known skill. Let me try to rewrite the script but I don't have a strong feeling on this part.